source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1235,6 +1235,122 @@ the pool's generation number then error handling MUST continue according to
 stale and the client MUST NOT update any topology state.
 (See `Why ignore errors based on CMAP's generation number?`_)
 
+Error handling pseudocode
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Application operations can fail in various places, for example:
+
+- A network error, network timeout, or command error may occur while
+  establishing a new connection. Establishing a connection includes the
+  MongoDB handshake and completing authentication (if configured).
+- A network error, network timeout may occur while reading or writing to an
```

Comment:
the comma here could be replace with "or" and would make this a little easier to read.

Reply:
Done.